build/scripts/DevEnv/NewDevEnv.psm1

```diff
@@ -388,9 +388,33 @@ function Get-CredentialForContainer($AuthenticationType) {
     }
 }
 
+<#
+    .Synopsis
+    Installs the AL extension in VSCode.
+
+    .Parameter ContainerName
+    The name of the container for which to install the extension.
+#>
+function Install-ALExtension($ContainerName) {
+    if (-not (Get-Command code -ErrorAction SilentlyContinue)) {
+        Write-Host "VSCode is not installed or 'code' is not in the PATH. See https://code.visualstudio.com/docs/setup/windows for installation instructions." -ForegroundColor Red
+        return
+    }
+
+    Write-Host "Installing VSCode extension..." -ForegroundColor Magenta
+    # Kill VSCode to avoid issues with the extension installation
+    Stop-Process -Name code -Force -ErrorAction SilentlyContinue
+
+    $vsixPath = Get-ChildItem "$($bcContainerHelperConfig.containerHelperFolder)\Extensions\$ContainerName\*.vsix" | Select-Object -ExpandProperty FullName
+    Write-Host "code --install-extension $vsixPath --force"
```

Comment:
As from description the point is to get latest version of AL Language.
Following things noticed:
 - If newer version is already installed it will downgrade.
 - If you have pre-release version, it will be overwritten by container Realease version of AL Language

Should there be checks for version newVersion > oldVersion?

Reply:
Thanks for the suggestion 👍 That's a good point. I'll remove the --force flag. If you have a newer version installed it'll just say: `A newer version of extension 'ms-dynamics-smb.al' vXX.XX is already installed.` but won't downgrade your AL Language version

Reply:
Thanks, that sounds good! Everything else looks good to me 😊